admin.go

```diff
@@ -0,0 +1,360 @@
+package sarama
+
+import "errors"
+
+// ClusterAdmin is the administrative client for Kafka, which supports managing and inspecting topics,
+// brokers, configurations and ACLs. The minimum broker version required is 0.10.0.0.
+// Methods with stricter requirements will specify the minimum broker version required.
+// This client was introduced in 0.11.0.0 and the API is still evolving.
+// You MUST call Close() on a client to avoid leaks
+type ClusterAdmin interface {
+	// Creates a new topic .This operation is supported by brokers with version 0.10.1.0 or higher.
```

Comment:
You don't need to specify the version requirement here, I prefer it at the end of the description where it already is for other methods.

Reply:
ok , will do that.

Reply:
done